pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [build-system]
-requires = ["setuptools", "wheel", "cmake"]
+requires = ["setuptools", "ninja; sys_platform != 'win32'", "wheel", "cmake"]
 build-backend = "setuptools.build_meta"
 
 [tool.cibuildwheel]
 build = "cp3*"
-skip = "*-win32 *-musllinux_i686 *-manylinux_i686"
+archs = ["auto64"]
+skip = "cp36-*"
```

Comment:
this can be replaced by introducing `python_requires='>=3.7'` in the `setup(...)` call of the `setup.py` file.

Reply:
you are missing a comma in `setup.py` at the end of the newly introduced line